.github/workflows/test.yml

```diff
@@ -7,8 +7,9 @@ jobs:
 
     runs-on: ubuntu-latest
     strategy:
+      max-parallel: 5
```

Comment:
Seems this is needed, otherwise Github erroneously interprets 3.10 as 3.1 ???

Reply:
Every time something isn't expressed in semver, sadness happens :frowning_man: